src/gitgraph.js

```diff
@@ -45,7 +45,8 @@
     case "vertical-reverse" :
       this.template.commit.spacingY *= -1;
       this.orientation = "vertical-reverse";
-      this.template.branch.labelRotation = 0;
+      this.template.branch.labelRotation =  !(_isNullOrUndefined(options, "template.branch.labelRotation")) ?
```

Comment:
I tend to avoid negated conditions for readability. Could you reverse this ternary?